README.md

```diff
@@ -62,7 +62,7 @@ Tendermint Core. You can subscribe [here](http://eepurl.com/gZ5hQD).
 
 | Requirement | Notes             |
 |-------------|-------------------|
-| Go version  | Go 1.18 or higher |
+| Go version  | Go 1.19 or higher |
```

Comment:
I would not make this change yet, and wait till we update the requirement in the build file.

Reply:
makes sense, too